Data-tranformations/fairgenomes/experiment.yarrrml.yaml

```diff
@@ -0,0 +1,22 @@
+prefixes:
+  ega.dataset: https://www.ebi.ac.uk/ega/datasets/
+  NCIT: http://purl.obolibrary.org/obo/NCIT_
+  OMIABIS: http://purl.obolibrary.org/obo/OMIABIS_
+  EDAM: https://bioportal.bioontology.org/ontologies/EDAM?p=classes&conceptid=
+  FIX: http://purl.obolibrary.org/obo/FIX_
+  GENEPIO: http://purl.obolibrary.org/obo/GENEPIO_
+
+
+mappings:
+  fairgenomes:
+    sources:
+      - [experiment.csv~csv] # I am going to name it like this 
+    s: ega.dataset:$(accession_id)
```

Comment:
`ega.dataset` -> `ega.experiments`

Reply:
There is no ega.experiments. It would be ega.runs ?

Reply:
You can access experiment information like this `https://metadata.ega-archive.org/experiments/{accession_id}`. 
E.g.: https://metadata.ega-archive.org/experiments/EGAX00002583861

For runs, there is similarly `https://metadata.ega-archive.org/runs/{accession_id}`